client/src/App.js

```diff
@@ -4,18 +4,36 @@ import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import Navbar from "./Components/Navbar";
 import User from "./Components/User";
 import CreateTask from "./Components/CreateTask";
+import UpdateTask from "./Components/UpdateTask";
+import Login from "./Components/Login";
+import Signup from "./Components/Signup";
+import UserHome from "./Components/UserHome";
+import UpdateTaskStatus from "./Components/UpdateTaskStatus";
+import ViewUsers from "./Components/ViewUsers";
+import LogOut from "./Components/LogOut";
+import './App.css'
 function App() {
   return (
     <>
 
   
       <BrowserRouter>
         <Routes>
+           
+
+          <Route path="/" element={<><Login/></>}/>
+          <Route path="/home" element={<><Home/> </>} />
+          <Route path="/signup" element={<><Signup/> </>} />
+          <Route path="/about" element={<><About/> </>} />
+          <Route path="/getUser/:id" element={<><User/> </>} />
```

Comment:
No consistency, In naming convention.
1. getUser
2. viewusers
3. updatetaskstatus